pkg/proxy/iptables/proxier.go

```diff
@@ -254,7 +256,7 @@ func NewProxier(ipt utiliptables.Interface,
 	healthzServer healthcheck.ProxierHealthUpdater,
 	nodePortAddresses []string,
 ) (*Proxier, error) {
-	if utilproxy.ContainsIPv4Loopback(nodePortAddresses) {
+	if !*disableLocalhostNodePorts && utilproxy.ContainsIPv4Loopback(nodePortAddresses) {
```

Comment:
this panics if `disableLocalhostNodePorts` is nil

https://go.dev/play/p/5XTq9MAOUQC

Reply:
sorry.. I miss it

Reply:
OK, I made a check for input parameter,  it's a good habit :)